docs/queries/read.md

```diff
@@ -173,6 +173,45 @@ tracks = await Track.objects.all()
 
 ```
 
+## iterate
+
+`iterate(*args, **kwargs) -> AsyncGenerator["Model"]`
+
+Return async iterable generator for all rows from a database for given model.
+
+Passing args and/or kwargs is a shortcut and equals to calling `filter(*args, **kwargs).iterate()`.
+
+If there are no rows meeting the criteria an empty async generator is returned.
+
+```python
+class Album(ormar.Model):
+    class Meta:
+        tablename = "album"
+        metadata = metadata
+        database = database
+
+    id: int = ormar.Integer(primary_key=True)
+    name: str = ormar.String(max_length=100)
+```
+
+```python
+await Album.objects.create(name='The Cat')
+await Album.objects.create(name='The Dog')
+# will Asynchronous Iterable Generator for All Rows Album Model:
```

Comment:
Please change the description in a comment to:
`will asynchronously iterate all Album models yielding one main model at a time from the generator`